### PR TITLE
Added air_cargo to planning.ipynb

### PR DESCRIPTION
**Pull request by Noumanmufc1**
I have added the `air_cargo` problem to planning.ipynb. @MrDupin  and @norvig Please review?

### CODE REVIEW

File: planning.ipynb [hunk 1]
```diff
@@ -302,30 +284,141 @@
...
+   "metadata": {},
+   "source": [
+    "# Air Cargo Problem:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "            Air Cargo problem involves loading and unloading of cargo and flying it from place to place. The problem can be with defined with three actions: Load, Unload and Fly."
```
Comment (ad71) [inline planning.ipynb]: I think you should remove the extra space before the sentence

File: planning.ipynb [hunk 2]
```diff
@@ -302,30 +284,141 @@
...
-   },
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "SyntaxError",
+     "evalue": "can't use starred expression here (<ipython-input-2-a15fe137afb2>, line 1)",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[1;36m  File \u001b[1;32m\"<ipython-input-2-a15fe137afb2>\"\u001b[1;36m, line \u001b[1;32m1\u001b[0m\n\u001b[1;33m    *psource(air_cargo)\u001b[0m\n\u001b[1;37m                       ^\u001b[0m\n\u001b[1;31mSyntaxError\u001b[0m\u001b[1;31m:\u001b[0m can't use starred expression here\n"
```
Comment (ad71) [inline planning.ipynb]: Correct the error in this cell

File: planning.ipynb [hunk 3]
```diff
@@ -302,30 +284,141 @@
...
+     ]
+    }
+   ],
+   "source": [
+    "*psource(air_cargo)"
```
Comment (ad71) [inline planning.ipynb]: Remove the asterisk before `psource` and execute the cell. This is the cause of the erroneous output.

**Comment by Noumanmufc1** [comment]
@ad71 thanks for pointing these out

**Pull request merged**